docs/configuration-format.md

```diff
@@ -0,0 +1,152 @@
+
+If more than one receiver is defined that listens on the same port it is treated as a fatal configuration error (note: it is valid for 2 receivers to listen on the same port number if they bind to different network interfaces).
+
+More than one pipeline can be associated with the same receiver definition. The definition of receiver is specific for each type however at the minimum it will include a port number and the network interface address to bind to (defaults to `127.0.0.1`).
+
+There are 2 possible configuration structure for a receiver. First structure is for receivers that support only one protocol:
+
+```yaml
+receivers:
+  <receiver name>:
+    endpoint: <network interface and port to bind to, address:port>
+    # other key/value pairs as needed by specific receiver type
+```
+
+For receivers that support more than one protocol the structure is the following:
+
+```yaml
+receivers:
+  <receiver name>:
+    protocols:
+      <protocol name 1>: # key is string, protocol name, unique
+        endpoint: <network interface and port to bind to, address:port>
+        # other key/value pairs as needed by specific receiver type
+      <protocol name 2>:
+        # settings for protocol 2
+      ...
+      <protocol name N>:
+        # settings for protocol N
+```
+
+There can be one or more `protocols` mappings, for example:
+
+```yaml
+receivers:
+  jaeger/external:
+    protocols:
+      thrift-tchannel:
+        endpoint: "127.0.0.1:14267”
+      thrift-http:
+        endpoint: "127.0.0.1:14268”
+```
+
+## Processors
+
+The configuration structure of a top-level processor is the following:
+
+```yaml
+processors:
+  <processor name>: # key is string, unique name of processor
+    # other key/value pairs as needed by specific processor type
+```
+
+The name should be in the form `type[/name]`, where `type` is the type of the processor (e.g. `batching`), and `name` suffix is optionally appended after a forward slash to ensure uniqueness of the full name. It is not allowed to define more than one processor with the same name.
+
+## Exporters
+
+Each exporter has a name which is defined by the key of the mapping under exporters top level-key. The name should be in the form `type[/name]`, where type is the type of the exporter (e.g. `otlp`), and `name` suffix is optionally appended after a forward slash to ensure uniqueness of the full name of the exporter. It is not allowed to define more than one exporter with the same name.
+
+Multiple pipelines can be associated with the same exporter. When this happens the data processed by these pipelines is directed to that exporter for further delivery.
+The definition of exporter is specific for each type however at the minimum it will include an endpoint string (which normally includes the destination address and port).
```

Comment:
Same here. The "endpoint" is not a requirement for an exporter.

Reply:
Removed